### PR TITLE
Updated partner management properties

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: partner-management-default.properties [hunk 1]
```diff
@@ -314,7 +314,9 @@ mosip.iam.post-logout-uri-param-key=post_logout_redirect_uri
 # end session endpoint in OIDC
 mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout
 # IDP claims mapping file url
-mosip.idp.claims-mapping-file-url= https://raw.githubusercontent.com/mosip/mosip-config/develop-v3/idp-claims-mapping.json
+mosip.idp.claims-mapping-file-url= https://raw.githubusercontent.com/mosip/mosip-config/master/identity-mapping.json
+# IDP amr-acr mapping file url
+mosip.idp.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/develop-v3/amr-acr-mapping.json
```
Comment (ghost) [inline partner-management-default.properties]: Pls change to `${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/amr-acr-mapping.json`

File: partner-management-default.properties [hunk 2]
```diff
@@ -314,7 +314,9 @@ mosip.iam.post-logout-uri-param-key=post_logout_redirect_uri
 # end session endpoint in OIDC
 mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout
 # IDP claims mapping file url
-mosip.idp.claims-mapping-file-url= https://raw.githubusercontent.com/mosip/mosip-config/develop-v3/idp-claims-mapping.json
+mosip.idp.claims-mapping-file-url= https://raw.githubusercontent.com/mosip/mosip-config/master/identity-mapping.json
```
Comment (ghost) [inline partner-management-default.properties]: Pls change to  `${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/identity-mapping.json`

File: partner-management-default.properties (hunk 1 above)
Comment (balaji-alluru) [reply partner-management-default.properties]: done anusha

File: partner-management-default.properties (hunk 2 above)
Comment (balaji-alluru) [reply partner-management-default.properties]: Done anusha

**Review by ghost — APPROVED** [review]

**Pull request merged**